app/views/notifications_mailer/session_notification.html.erb

```diff
@@ -0,0 +1,59 @@
+<div class="container">
+  <% starts_in = minutes_between(DateTime.now, @session.starts_at) %>
+  <% if starts_in > 1  && starts_in < 60 %>
+    <h1 class="title">Your bookmarked session is starting in <%= starts_in %> minutes </h1>
+  <% elsif starts_in <= 1 %>
+    <h1 class="title">Your bookmarked session is starting now</h1>
+  <% else %>
+    <h1 class="title">Your bookmarked session is starting soon </h1>
+  <% end %>
```

Comment:
The `title` method in the `SessionReminderNotifier` class might be useful for defining the title. If it doesn't fit the purpose, I suggest extracting this logic into the `SessionReminderNotifier` class to maintain consistency.